src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,90 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+
+    @BeforeAll
+    static void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    public void register_validUser_ok() {
+        User user = new User("validLogin", "password123", 20);
+        User registeredUser = registrationService.register(user);
+        assertEquals(user, registeredUser);
```

Comment:
This is a good test, but it would be even better if you also checked that the user was added to the storage. You could do this by asserting that `Storage.people.contains(user)` is true after the registration.